src/CZ.Azure.FileExchange/Client/styles/components/_button.scss

```diff
@@ -0,0 +1,33 @@
+@use '../abstracts/variables';
+@use '../abstracts/mixins';
+
+button{
```

Comment:
this is the upload button? can you maybe rename this?

Reply:
okay nevermind. that means a different button

Reply:
This is the generic styling for buttons